### PR TITLE
Add style for the page listing all the blog posts

### LINKED ISSUE #74: Re-implement list of posts (blog template)
It looks like this:

Screen Shot 2021-03-31 at 23 26 30

We can deal with the header and pagination in other issues.

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #74 

---

The header has not been updated, only the "main" content. The xxl breakpoint has been updated to respect the max content width, which makes things easier. It's fully responsive. I added some more comments below. It looks quite ok I would say, and we can still iterate later (once we have feedback from UX/Product)

Deployed at: https://addons-blog-preview-willdurand.vercel.app/blog/

### CODE REVIEW

File: src/assets/css/blog.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+$article-padding: 19px;
+
+$max-featured-image-height: 236px;
+
+$title-color: #291c48;
+
+$date-color: #54556b;
+$date-font-size: 12px;
+
+$excerpt-margin: 13px;
```
Comment (willdurand) [inline src/assets/css/blog.scss:12]: All these variables are "local" (or specific to that page), which is why they are there and not in the `variables.scss` file.

File: src/assets/css/blog.scss [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+.blog-articles {
+  display: grid;
+  grid-gap: 22px;
```
Comment (willdurand) [inline src/assets/css/blog.scss:16]: I used Figma to find the different sizes, which is why we have 22, 19, etc.

File: src/assets/css/inc/variables.scss [hunk 3]
```diff
@@ -27,14 +27,6 @@ $heading-md-line-height: 44px;
 // Instead of adding a rule for only mobile, add a mobile rule and override it
 // with `@include respond-to(m) { [...] }`.
 $breakpoints: (
-  m-only: only screen and
```
Comment (willdurand) [inline src/assets/css/inc/variables.scss:30]: This isn't used and I'd like to avoid too many breakpoints.

File: src/content/blog.njk [hunk 4]
```diff
@@ -1,33 +1,17 @@
...
 
 {% block content %}
-	{% set allArticles = pagination.items %}
+<ul class="blog-articles">
+  {%- for post in posts -%}
+    {%- set postFeaturedImage = media | mediaGetMediumURL(post.featuredImage) -%}
+    {%- set postPermalink = post.permalink -%}
+    {%- set postTitle = post.title -%}
+    {%- set postDate = post.date | readableDate -%}
+    {%- set postExcerpt = post.excerpt -%}
```
Comment (willdurand) [inline src/content/blog.njk:12]: All of this is needed to be able to share a common template (include) for both the actual blog content and the WP theme 🚀

File: src/includes/blog-article.njk [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<li class="blog-article">
```
Comment (muffinresearch) [inline src/includes/blog-article.njk:1]: Any reason for using tabs in this file? 🤔

File: src/includes/blog-article.njk [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+	</p>
+
+	<div class="blog-article-excerpt">
+		{{ postExcerpt | safe }}
+
+		<p class="blog-article-read-more">
+			<a class="blog-article-read-more-link" href="{{ postPermalink }}">
+				Read more
+			</a>
+			&nbsp;<span class="blog-article-read-more-arrow">▸</span>
```
Comment (muffinresearch) [inline src/includes/blog-article.njk:21]: Could this be done with content via CSS instead since it's primarily presentational?

**Review by muffinresearch — APPROVED** [review]
r+wc 

Looks good. Might be nice to have a default (or a set of random default) placeholder image(s) if there's no featured image for the post.

File: src/includes/blog-article.njk (hunk 5 above)
Comment (willdurand) [reply src/includes/blog-article.njk:1]: mm nope, I'll fix that.

File: src/includes/blog-article.njk (hunk 6 above)
Comment (willdurand) [reply src/includes/blog-article.njk:21]: I'll look into that

**Comment by willdurand** [comment]
> Looks good. Might be nice to have a default (or a set of random default) placeholder image(s) if there's no featured image for the post.

Do you mean in WordPress itself? I am wondering how to deal with "no featured image" here. We should probably not have the broken image in the first place, but then what? No image? Or a fat blue rectangle?

**Pull request merged**